### ISSUE TITLE
Usability testing for Policy Viewer

### ISSUE THREAD

**Issue by yowill** [18F/omb-eregs#248]
If we can get everything finished by Wednesday, build out the convo guide and line up folks for testing

**Comment by nicoleslaw** [comment]
Blocked by #240 and #239

**Comment by nicoleslaw** [comment]
In touch with a user— we are talking about doing a usability test next Tuesday. I will set up a spreadsheet for the testing plan.

**Comment by nicoleslaw** [comment]
Plan doc is here: https://docs.google.com/spreadsheets/d/18-9gpJCZeNp2gzLXrldbgNLKpoewhYP4cioeJ0SOYNQ/edit#gid=0

@micahtaylor Are we planning to use Invision this round, or only the working prototype?

**Comment by micahtaylor** [comment]
Just now seeing this @nicoleslaw, we answered this by saying the working prototype for the record.

**Comment by nicoleslaw** [comment]
This is done—but I would like to talk through what we do with research after testing.

**Comment by theresaanna** [comment]
@nicoleslaw to create a new issue for follow up work.
